daemon/lib/swaps/40swap.go

```diff
@@ -0,0 +1,88 @@
+package swaps
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/40acres/40swap/daemon/api"
+	"github.com/40acres/40swap/daemon/database/models"
+)
+
+type FourtySwapClient struct {
+	client *api.Client
+}
+
+func NewFourtySwapClient(endpoint string) (*FourtySwapClient, error) {
+	client, err := api.NewClient(endpoint)
+	if err != nil {
+		return nil, err
+	}
+
+	return &FourtySwapClient{
+		client: client,
+	}, nil
+}
+
+func chainToDtoChain(chain models.Chain) (api.SwapOutRequestDtoChain, error) {
+	switch chain {
+	case models.Bitcoin:
+		return api.BITCOIN, nil
+	case models.Liquid:
+		return api.LIQUID, nil
+	default:
+		return api.BITCOIN, fmt.Errorf("invalid chain: %s", chain)
+	}
+}
+
+func (f *FourtySwapClient) CreateSwapOut(ctx context.Context, swapReq CreateSwapOutRequest) (*SwapOutResponse, error) {
+	chain, err := chainToDtoChain(swapReq.Chain)
+	if err != nil {
+		return nil, err
+	}
+
+	body := api.SwapOutControllerCreateSwapJSONRequestBody{
+		Chain:        chain,
+		ClaimPubKey:  swapReq.ClaimPubKey,
+		InputAmount:  float32(swapReq.Amount.InexactFloat64()),
```

Comment:
are we going to round?

Reply:
I think this will be better handled once we have the money type being used here

Reply:
using money now, will ensure max of 8 decimals